dotcom-rendering/src/components/marketing/banners/designableBanner/settings.ts

```diff
@@ -33,10 +33,13 @@ export interface HighlightedTextSettings {
 }
 
 export interface TickerStylingSettings {
-	textColour: string;
+	text: string; //deprecated
```

Comment:
can this model (`TickerStylingSettings`) be deleted now that we're using the source component?

Reply:
great idea - I've reworked it to do so